pkg/trace/api/pipeline_stats.go

```diff
@@ -112,7 +115,14 @@ func (m *multiDataStreamsTransport) RoundTrip(req *http.Request) (rresp *http.Re
 	}
 	if len(m.targets) == 1 {
 		setTarget(req, m.targets[0], m.keys[0])
-		return m.rt.RoundTrip(req)
+		rresp, rerr = m.rt.RoundTrip(req)
+		if rerr != nil {
+			log.Error(rerr)
```

Comment:
does the error have enough context here?

Reply:
done